clang/compiler-course/baranov_a/baranov_ast.cpp

```diff
@@ -0,0 +1,120 @@
+#include "clang/AST/ASTConsumer.h"
+#include "clang/AST/RecursiveASTVisitor.h"
+#include "clang/Frontend/CompilerInstance.h"
+#include "clang/Frontend/FrontendPluginRegistry.h"
+#include "llvm/Support/raw_ostream.h"
+#include <string>
+namespace {
```

Comment:
For readability, it is better to separate an empty line
```suggestion
#include <string>

namespace {
```